package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "simple-chat",
+  "version": "1.0.0",
+  "description": "A simple chat app using Angular and socket.io",
+  "main": "web_server.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "node web_server.js"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/techpool/simple-chat.git"
```

Comment:
URL should be _github.com/mbrandau/simple-chat.git_ (I know it's generated by npm :wink:)

Reply:
Fixed.